hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -1363,24 +1370,17 @@ public Response saveAllJobs(@QueryParam("user") String user,  @QueryParam("token
             log.trace("Save all jobs exception", ex);
             return Response.ok("{\"operation\":\"failed: " + ex.toString() + "\"").build();
         }
-
     }
 
     private static Response buildServerError(Exception exception) {
-        log.warn("", exception);
+        log.error("Error in kick job: {}", Throwables.getStackTraceAsString(exception));
```

Comment:
Do not use `Throwables.getStackTraceAsString` to parameterize a logging message. The correct methods on slf4j logger for logging an exception are:
```
public void warn(String msg, Throwable t);
public void error(String msg, Throwable t);
```